hearth/lib/hearth/refreshing_identity.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module Hearth
+  # Resolves an identity from identity properties.
+  module RefreshingIdentity
```

Comment:
The naming here is a little confusing I think.  This seems like it should be the `RefreshingIdentityResolver`

Reply:
Refreshing Identity Resolver is already a class that we have now.